DecipherNow/gm-fabric-dashboard
# chore: Add new codeclimate badges

Resolves #1212 

## Proposed Changes

  - Add new awesome badges

## How to Test

  - In the GitHub Code pane, select the 1212--codeclimate-badges branch and then scroll down to see the new README with new badges.

![image](https://user-images.githubusercontent.com/8700322/33910835-2069c0fc-df5e-11e7-81a1-b457b764d3de.png)
